packages/core/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -9,43 +9,38 @@
  */
 
 import { ApiPromise, WsProvider } from '@polkadot/api'
-import { RegistryTypes } from '@polkadot/types/types'
 import Blockchain from '../blockchain/Blockchain'
+import MASHNET_TYPES from './types/mashnet'
+import PARACHAIN_TYPES from './types/parachain'
 
 export const DEFAULT_WS_ADDRESS =
-  process.env.DEFAULT_WS_ADDRESS || 'ws://127.0.0.1:9944'
+  process.env.DEFAULT_WS_ADDRESS || 'ws://127.0.0.1:9977'
```

Comment:
why this change?

Reply:
Parachains have two ws ports. 9944 is for the relay chain, 9977 is for the parachain.

We connect to the parachain...